Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1455,6 +1480,16 @@ extension BrowserViewController: TabManagerDelegate {
 }
 
 extension BrowserViewController: WKNavigationDelegate {
+    func webView(webView: WKWebView, didCommitNavigation navigation: WKNavigation!) {
```

Comment:
Even if `didCommitNavigation` works, `didFinishNavigation` seems more appropriate, I think, since it's called after the main frame load completes. Logically, that's what we would want to wait for before setting the UA for the next page.